src/js/models/layer.js

```diff
@@ -134,12 +134,12 @@ define(function (require, exports, module) {
         /**
          * @type {Immutable.List.<Shadow>}
          */
```

Comment:
I don't think you need to wait until after the release, though it might be better to submit a separate PR for that particular cleanup to make the review easier.